### PR TITLE
Added a section for debugging webui issues

### PR DESCRIPTION
**Pull request by jwmatthews**

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -92,6 +93,29 @@ The migration debug tree can be viewed and traced by querying specific label sel
...
 See [Viewing migration custom resources](https://docs.openshift.com/container-platform/4.5/migration/migrating_3_4/troubleshooting-3-4.html#migration-viewing-migration-crs_migrating-3-4) for more information.
 
+# Debugging WebUI
+
+## Web browser sees 'A certificate error has occurred....'
+
+* Example of error message displayed in web browser
+
+    ```
+        A certificate error has occurred, likely due to the usage of self signed certificates in one of the clusters. Please try to visit the failed url and accept the CA: https://api.ocp.foo.com:6443/.well-known/oauth-authorization-server
```
Comment (apinnick) [inline troubleshooting.md:103]: John, if this error message is accurate, it needs some editing. Can you send me a link to a UI mockup or a mig-ui PR?

Comment (jwmatthews) [reply troubleshooting.md:103]: Here is the code in mig-ui for the error: 
https://github.com/konveyor/mig-ui/blob/master/src/app/auth/CertErrorComponent.tsx#L16-L25

If you have suggestions to improve let us know.  

Note on technical details....we are limited with what we can detect so this error message is a little vague and it's a partial guess.  This is typically shown for a cert failure when self signed certs are involved, yet other edge cases could cause it as well when client communication is impacted.


As to the downstream docs and [CA bundle](https://docs.openshift.com/container-platform/4.5/migration/migrating_3_4/migrating-applications-with-cam-3-4.html#creating-ca-bundle_migrating-3-4), those steps are for adding the CA bundle text to creation of a resource.  

We could consider expanding the downstream docs to provide instructions for optionally configuring the CA bundle in the web browsers trust store or explaining more of what this error could mean, filed https://issues.redhat.com/browse/MIG-358 to track.

Comment (apinnick) [reply troubleshooting.md:103]: I created a mig-ui issue for the error message: https://github.com/konveyor/mig-ui/issues/1043

Comment (apinnick) [reply troubleshooting.md:103]: I will accept this PR and then edit it myself.

**Pull request merged**